packages/kbn-server-route-repository/kibana.jsonc

```diff
@@ -1,5 +1,5 @@
 {
   "type": "shared-common",
   "id": "@kbn/server-route-repository",
-  "owner": "@elastic/apm-ui"
+  "owner": ["@elastic/obs-knowledge-team", "@elastic/obs-ux-management-team"]
```

Comment:
Any reason why this should also be owned by the obs-ux-management team? Is an approving review of all teams or one of the listed teams needed?

Reply:
@dgieselaar a review of only one of the teams (not both) is required (as per the [GH docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)).

I understand this is used by APM, profiling, AI assistant and others, so the idea was to have both teams as owner, so either could review.  Are you suggesting it should have another team too (such as `obs-ux-infra_services-team`)?